cumulus/polkadot-parachain/src/chain_spec/coretime.rs

```diff
@@ -0,0 +1,285 @@
+// Copyright Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::chain_spec::GenericChainSpec;
+use cumulus_primitives_core::ParaId;
+use sc_chain_spec::ChainSpec;
+use std::str::FromStr;
+
+/// Collects all supported Coretime configurations.
+#[derive(Debug, PartialEq)]
+pub enum CoretimeRuntimeType {
+	// Live
+	Rococo,
+	// Local
+	RococoLocal,
+	// Benchmarks
+	RococoDevelopment,
+
+	// Live
+	Westend,
+	// Local
+	WestendLocal,
+	// Benchmarks
+	WestendDevelopment,
+}
+
+impl FromStr for CoretimeRuntimeType {
+	type Err = String;
+
+	fn from_str(value: &str) -> Result<Self, Self::Err> {
+		match value {
+			rococo::CORETIME_ROCOCO => Ok(CoretimeRuntimeType::Rococo),
+			rococo::CORETIME_ROCOCO_LOCAL => Ok(CoretimeRuntimeType::RococoLocal),
+			rococo::CORETIME_ROCOCO_DEVELOPMENT => Ok(CoretimeRuntimeType::RococoDevelopment),
+			westend::CORETIME_WESTEND => Ok(CoretimeRuntimeType::Westend),
+			westend::CORETIME_WESTEND_LOCAL => Ok(CoretimeRuntimeType::WestendLocal),
+			westend::CORETIME_WESTEND_DEVELOPMENT => Ok(CoretimeRuntimeType::WestendDevelopment),
+			_ => Err(format!("Value '{}' is not configured yet", value)),
+		}
+	}
+}
+
+impl CoretimeRuntimeType {
+	pub const ID_PREFIX: &'static str = "coretime";
+
+	pub fn load_config(&self) -> Result<Box<dyn ChainSpec>, String> {
+		match self {
+			CoretimeRuntimeType::Rococo => Ok(Box::new(GenericChainSpec::from_json_bytes(
+				&include_bytes!("../../../parachains/chain-specs/coretime-rococo.json")[..],
+			)?)),
+			CoretimeRuntimeType::RococoLocal => Ok(Box::new(rococo::local_config(
+				rococo::CORETIME_ROCOCO_DEVELOPMENT,
+				"Rococo Coretime Local",
```

Comment:
Should this be:

```suggestion
			CoretimeRuntimeType::RococoLocal => Ok(Box::new(rococo::local_config(
				rococo::CORETIME_ROCOCO_LOCAL,
				"Rococo Coretime Local",
```

and same for westend below?

Reply:
Thanks, that's some copy pasta